chipsets.h

```diff
@@ -505,82 +521,87 @@ template <uint8_t DATA_PIN, EOrder RGB_ORDER = RGB>
 class PL9823Controller : public ClocklessController<DATA_PIN, 3 * FMUL, 8 * FMUL, 3 * FMUL, RGB_ORDER> {};
 
 #else
+    
+// Similar to NS() macro, this calculates the number of cycles for
+// the clockless chipset (which may differ from CPU cycles)
+#define C_NS(_NS) (((_NS * ((CLOCKLESS_FREQUENCY / 1000000L) + 999) / 1000)
```

Comment:
Next time - please make sure your stuff compiles - and compiles for other platforms before you submit a PR -- you broke compilation for pretty much every platform because of unbalanced parens here.

Reply:
You're right.  This was a mistake on my part, in that I didn't have a setup to compile multiple platforms.  I hope it's obvious that it wasn't my intent to break the build.

Wondering if there's travis support for Arduino...  Yes!  (See https://github.com/adafruit/travis-ci-arduino and https://learn.adafruit.com/the-well-automated-arduino-library/repository).

Would there be interest in Travis CI builds for FastLED?  Not a cure-all, but a step in the right direction....

Reply:
I'll just see if I can get it going, and if so, I'll offer it up.  Should be easier than adding support for a new chip.